src/jit/gentree.h

```diff
@@ -1138,18 +1130,18 @@ struct GenTree
         return (gtOper == GT_LEA);
     }
 
-    bool OperIsBlkOp() const;
-    bool OperIsCopyBlkOp() const;
-    bool OperIsInitBlkOp() const;
+    bool OperIsBlkOp();
+    bool OperIsCopyBlkOp();
+    bool OperIsInitBlkOp();
     bool OperIsDynBlkOp();
 
-    static
-    bool OperIsBlk(genTreeOps gtOper)
+    static bool OperIsBlk(genTreeOps gtOper)
     {
-        return (gtOper == GT_OBJ);
+        return ((gtOper == GT_BLK) || (gtOper == GT_OBJ) || (gtOper == GT_DYN_BLK) || (gtOper == GT_STORE_BLK) ||
+                (gtOper == GT_STORE_OBJ) || (gtOper == GT_STORE_DYN_BLK));
     }
 
-    bool OperIsBlk() const
+    bool OperIsBlk()
     {
```

Comment:
I wouldn't think so. I had to change some methods that pertain to the blk ops/assignments because they had to check some things that aren't const, but this one shouldn't be a problem. I'll change it.

---
In reply to: [77247543](https://github.com/dotnet/coreclr/pull/7027#discussion_r77247543) [](ancestors = 77247543)